workflow-controller/controller-service/src/main/java/io/hyscale/controller/Converters/ProfileConverter.java

```diff
@@ -46,33 +47,32 @@ public String getFilePattern() {
 
 
     @Override
-    public String getReferenceSchema() {
-        return PROFILE_SCHEMA_FILE;
+    public HyscaleSpecType getReferenceSchemaType() {
+        return HyscaleSpecType.ENVIRONMENT;
```

Comment:
This getRefereceSchemaType should be 
service spec
profile

why environment ?

Reply:
yes